tests/integration-tests/tests/create/test_create/test_create_wrong_os/pcluster.config.ini

```diff
@@ -0,0 +1,19 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = slurm
```

Comment:
Rather than hardcoding I would expose this in the config as done in the other tests. Same applies elsewhere

Reply:
I will change it